R/indentation_linter.R

```diff
@@ -0,0 +1,163 @@
+#' Check that indentation is consistent
+#'
+#' @param indent Block indentation level, used for multi-line code blocks (`{ ... }`), function calls (`( ... )`) and
```

Comment:
I think it's not clear what this means from this description, maybe at least mention the default is `2` following the tidyverse recommendation?